test_conformance/non_uniform_work_group/TestNonUniformWorkGroup.cpp

```diff
@@ -482,7 +539,9 @@ int TestNonUniformWorkGroup::prepareDevice () {
   if(_localSize_IsNull == false)
     calculateExpectedValues();
 
-  std::string buildOptions = BUILD_CL_STD_2_0;
+  std::string buildOptions = (get_device_cl_version(_device) == Version(3, 0))
```

Comment:
I am not sure if this is needed. 
@bashbaug do we need to change OpenCL C language version in this and other cases ?

Reply:
Yes, I believe so.  I didn't do an exhaustive check, but I do see that the kernel string for this test includes the `get_enqueued_local_size()` built-in function, which requires OpenCL C 2.0 or OpenCL C 3.0 and was not in earlier versions of OpenCL C.

Reply:
With #879 , the selection could be completely omitted.